dev/release/verify-release-candidate.sh

```diff
@@ -326,7 +326,13 @@ install_nodejs() {
       PROFILE=/dev/null bash
     [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
 
-    nvm install --lts
+    min_glibc_version_for_node_18_or_greater=2.25
+    glibc_version=$(ldd --version | grep ldd | rev | cut --delimiter=' '  -f1 | rev)
+    if [ -n "${glibc_version}" -ge "${min_glibc_version_for_node_18_or_greater}" ]; then
+      nvm install --lts
+    else
+      nvm install --lts=Gallium
+    fi
```

Comment:
This code is also used by macOS. So using `ldd` isn't a good idea.
How about always using `nvm install --lts=Gallium` or detecting CentOS 7 (`[ -f /etc/centos-release-upstream ]`) instead?